src/routes/+page.ts

```diff
@@ -27,25 +33,59 @@ async function requestProjectMetadata(project: Project): Promise<ProjectMetadata
     };
 }
 
+async function requestUndecidedPapers(project: Project): Promise<PaperInfos[]> {
+    const projectController = BackendController.getInstance().project(project.id);
+    const currentStage: number = await projectController.getCurrentStage();
+    const latestStage: number = await projectController.getStageCount();
+    const allUndecidedPapers: PaperInfos[] = [];
+
+    const numberOfRequiredReviews = await projectController
+        .get()
+        .then((project) => project.reviewDecisionMatrix.numberOfReviewers);
```

Comment:
Here, we already have the whole project as an object, so we can just use the property directly
```suggestion
    const numberOfRequiredReviews = project.reviewDecisionMatrix.numberOfReviewers;
```

Reply:
Sorry, missed it. Thank you, I will change it.